docs/Upgrading.md

```diff
@@ -25,6 +25,13 @@ Now install that version of `react-native` in your project with `npm install --s
 ```sh
 $ npm install --save react-native@X.Y 
 # where X.Y is the semantic version you are upgrading to
+npm WARN peerDependencies The peer dependency react@~R included from react-native...
+```
+
+If you saw a warning about the peerDependency, also install `react`:
```

Comment:
I'm not sure about being so specified. If `react` is upgraded, it could lead to further unmet dependency (in case of other dependencies declare `react` as peer, like React addons for example).

I'd be more evasive like "fix any unmet peer dependency requirements".